src/System.IO.FileSystem/tests/packages.config

```diff
@@ -1,7 +1,8 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="System.Diagnostics.Debug" version="4.0.10-beta-22512" targetFramework="portable-net45+win+wpa81" />
   <package id="System.IO" version="4.0.10-beta-22512" targetFramework="portable-net45+win+wpa81" />
+  <package id="System.IO.FileSystem.Primitives" version="4.0.0-beta-22605" targetFramework="portable-net45+win" />
```

Comment:
No, Nuget package is fine, but please use the same build as the rest of the project: 22512.  Also can you make sure targetFramework is the same?  It's ok to copy/paste another package element (Nuget package manager isn't working well with our build time package resolution at the moment).